Bulky.DataAccess/Repository/ProductRepository.cs

```diff
@@ -20,7 +20,23 @@ public ProductRepository(ApplicationDbContext db) : base(db)
 
         public void Update(Product obj)
         {
-            _db.Products.Update(obj);
+            var objFromDb = _db.Products.FirstOrDefault(u => u.Id == obj.Id);
+            if (objFromDb != null)
+            {
+                objFromDb.Title = obj.Title;
```

Comment:
Task for this week will be to implement automapper into your project and see how it simplifies this block of code